### PR TITLE
Adding isPartialFacadeAssembly=true for System.Numerics.Vectors

### PR DESCRIPTION
**Pull request by venkat-raman251**
@weshaggard  : I missed this condition here as well.

### CODE REVIEW

**Comment by mellinoe** [comment]
Should you commit this as yourself, or did we want it as dotnet-bot for some reason?

**Comment by venkat-raman251** [comment]
@mellinoe : Yeah, i was told we want porting to be done as dotnet-bot. This was something i missed in the porting, so stuck to using dotnet-bot.

**Comment by weshaggard** [comment]
LGTM. I'm fine with using dotnet-bot for this porting efforts.

File: src/System.Numerics.Vectors/src/project.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     },
```
Comment (weshaggard) [inline src/System.Numerics.Vectors/src/project.json]: could you change dnxcore to dotnet5.4

Comment (venkat-raman251) [inline src/System.Numerics.Vectors/src/project.json]: @weshaggard : changed.

**Comment by weshaggard** [comment]
LGTM merge when CI is green.

**Pull request merged**